recipes/nlohmann_json/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "nlohmann_json" %}
+{% set version = "3.1.2" %}
+{% set sha256 = "e8fffa6cbdb3c15ecdff32eebf958b6c686bc188da8ad5c6489462d16f83ae54" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/nlohmann/json/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
I am not sure that we need the pure `c` compiler (as well as the c compiler section in the configuration), but someone from conda-forge might have some idea about the best practice.

Reply:
Maybe @jakirkham can comment

Reply:
No harm in including it, but I agree it's probably not needed.